sdk/core/src/headers/mod.rs

```diff
@@ -3,6 +3,8 @@ mod utilities;
 use http::request::Builder;
 pub use utilities::*;
 
+pub const MS_DATE: &str = "x-ms-date"; //=> [String] }
```

Comment:
I'm not sure what the comment is supposed to signify. Is it needed?

Reply:
No, it's not. 
Also, now that we've embraced the use of your `AddAsHeaders` trait I'm also wondering if it makes sense to have these constants scattered around different mods. We could simply hardcode the correct value in the `AddAsHeaders` implementation (where it will be used). 

What do you think?